static/src/stylesheets/module/facia/_mixins.scss

```diff
@@ -22,15 +22,15 @@ $breakpoints: (
             margin-left: $gs-gutter;
             margin-right: $gs-gutter;
         }
-        @include mq(containerWidestMobile) {
```

Comment:
This as agreed with Stephan and Zef. I'll double check, when Zef is doing his review ...